documentation/doc-Migration_Toolkit_for_Virtualization/modules/migrating-virtual-machines-cli.adoc

```diff
@@ -136,14 +140,15 @@ $ cat << EOF | oc apply -f -
 apiVersion: virt.konveyor.io/v1alpha1
 kind: Migration
 metadata:
-  name: migration-001
+  name: migration_name
   namespace: {namespace}
 spec:
   plan:
-    name: test-migration-plan
+    name: <plan_name> <1>
```

Comment:
Maybe use the same, <plan_name> in the Plan CR (line 96  in this diff).

Reply:
Good idea. I was not sure what the 'metadata' part meant. Updated line 96